motionSetPointsApp/src/setPoint.cpp

```diff
@@ -140,12 +147,17 @@ void loadFile(FileIOInterface *fileIO, const char *fname, const char *env_fname,
 	LookupTable& table = getTable(env_fname);
     epicsGuard<epicsMutex> _lock(g_lock); // need to protect write access to map	
 	table.rows.clear();
-    table.error.clear();
-    // "fname" is the full path to the file, and as we have a limit of 40 characters we extract just the file name.
-    table.fileName = strrchr(fname, '/') ? strrchr(fname, '/') + 1 : fname;
+    table.error = "No error";
+    // "fname" is the full path to the file, and as there is a limit of 40 characters, extract just the file name.
+    std::string fnameString = fname;
+    size_t lastSeparator = fnameString.find_last_of("\\/", std::string::npos, 2);
```

Comment:
```suggestion
    size_t lastSeparator = fnameString.find_last_of("\\/");
```
See example: https://cplusplus.com/reference/string/string/find_last_of/ for example of splitting path to get file name